accessReviewScheduleDefinition docs refresh (#12447)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/removeAccessApplyAction.md

---
title: "removeAccessApplyAction resource type"

> description: "Type of accessReviewApplyAction that indicates removing access to a resource of those denied in an access review of that resource."

Reviewer: ```suggestion
description: "Indicates that access is removed from a resource for those denied in an access review of that resource."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# removeAccessApplyAction resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> removeAccessApplyAction is a kind of [accessReviewApplyAction](../resources/accessreviewapplyaction.md) that indicates removing access to a resource of those denied in an access review of that resource. This is the default option chosen for [accessReviewApplyAction](../resources/accessreviewapplyaction.md) if an option is not passed in.

Reviewer: ```suggestion
Remove access to a resource of those denied in an access review of that resource. This is the default option chosen for [accessReviewApplyAction](../resources/accessreviewapplyaction.md) if an option is not passed in.
```
